lib/modules/outlet/devices/cuco.plug.co1.js

```diff
@@ -0,0 +1,48 @@
+const OutletDevice = require('../OutletDevice.js');
+const Properties = require('../../../constants/Properties.js');
+const Actions = require('../../../constants/Actions.js');
+const Constants = require('../../../constants/Constants.js');
+const PropFormat = require('../../../constants/PropFormat.js');
+const PropUnit = require('../../../constants/PropUnit.js');
+const PropAccess = require('../../../constants/PropAccess.js');
+
+
+class CucoPlugCp1 extends OutletDevice {
```

Comment:
You forgot to change the class name here. The export on the bottom is correct but it will not work because of this...